bot/commands/commands.py

```diff
@@ -93,12 +95,27 @@ async def botrole(command, msg, user, channel, *args, **kwargs):
 @register('chal')
 async def challonge(command, msg, user, channel, *args, **kwargs):
     if len(msg.split(' ')) < 2:
```

Comment:
needs to be < 1. right now it can't execute without a bracket provided so it can never automatically grab the bracket from the DB

Reply:
msg.split will always be 1 with an empty string. you have to check if the value is less than 2